extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -593,6 +593,15 @@ export const targetPlatformToVersion: Map<string, string> = new Map<string, stri
 	[SqlTargetPlatform.sqlDW, 'Dw']
 ]);
 
+export const serverVersionToTargetPlatform: Map<number, SqlTargetPlatform> = new Map<number, SqlTargetPlatform>([
```

Comment:
let's add "onPrem" to the name of the map so it's clear the Azure versions aren't here